mpas_analysis/obs/observational_datasets.xml

```diff
@@ -55,6 +55,7 @@
       - ftp://ftp.cgd.ucar.edu/archive/SSTICE/MODEL.SST.HAD187001-198110.OI198111-201712.nc
     </dataUrls>
     <preprocessing>
+      None
```

Comment:
This is fine, though I think it's fine to just leave it blank if there is no preprocessing.